style/americana.js

```diff
@@ -104,6 +125,11 @@ bridgeLayers.forEach((layer) =>
 americanaLayers.push(
   //The labels at the end of the list have the highest priority.
```

Comment:
For my understanding of maplibre gl — just as later layers paint on top of previous ones and obscure them, the later text layers will take priority and suppress text/symbol layers before them?

Reply:
Hmm, I would have assumed the earlier layers take priority since they're already placed.